compiler/AST/type.cpp

```diff
@@ -550,6 +550,12 @@ addDeclaration(AggregateType* ct, DefExpr* def, bool tail) {
       fn->addFlag(FLAG_METHOD_PRIMARY);
     }
   }
+
+  if (VarSymbol* var = toVarSymbol(def->sym)) {
+    // Identify VarSymbol as class/record member.
+    var->makeField();
```

Comment:
Is this intended to be a question? If so, the answer is no. Or more specifically, it is not used by anything outside the docs() pass.